uwblueprint/planet-read
# Soft Delete Stories

## Notion ticket link



[Ticket ](https://www.notion.so/uwblueprintexecs/a658c933d18c48d4b43fd99bda2bd3b4?v=ba22ac93d09040ff8219c60ac7caabdb&p=69f5f0a450324aad84df236d4336d1f3)


## Implementation description

- Create new StoryAll and StoryContentAll model to add new column `is_deleted`
- Create new view for Story and StoryContent 
- Backend Changes: 
- Create mutation `SoftDeleteStory`
- Frontend: 
- Call new mutation in `ManageStoryPage`



## Steps to test

1. Login as admin 
2. Go to story page and delete story 
3. Ensure story and story translations are no longer present and you are redirected to the homepage 



## What should reviewers focus on?

- The soft delete story flow makes sense 
- All related StoryContent, StoryTranslations are soft-deleted as well 

## Checklist
